### ISSUE TITLE
Improve loading in ROSE UI list views (interaction/ comments / extracts)

### ISSUE THREAD

**Issue by dharma-guardian** [secure-software-engineering/rose#220]
After #211, I experienced something similar under Facebook Data/Interactions. The loading took around 2 seconds and left me wondering. I have 286 interactions. I don't know, if pagination is necessary. I consider this low priority.

Here is a snapshot of my current ROSE Data for testing: 
[rose-data.txt](https://github.com/secure-software-engineering/rose/files/90714/rose-data.txt)

<img width="440" alt="Screenshot ROSE UI Interactions Profile" src="https://cloud.githubusercontent.com/assets/1899308/12331181/ca4786be-bae8-11e5-9715-229678c2e9af.png">

**Comment by iamolivinius** [comment]
Please add CPU profile.

**Comment by dharma-guardian** [comment]
There your go: [CPU-20160114T183743.cpuprofile.txt](https://github.com/secure-software-engineering/rose/files/90754/CPU-20160114T183743.cpuprofile.txt)

**Comment by iamolivinius** [comment]
This is ember related. Rendering a large number of complex components takes time, pagination would improve initial rendering performance.

**Comment by dharma-guardian** [comment]
I hope so, but could it be that Ember renders pagination completely, instead of using lazy loading? Had the impression with your implementation in #211 .

**Comment by iamolivinius** [comment]
It retrieves log data completely but renders only 1 page at a time (20
elements per page). Lazy loading is handy if you have limited
bandwidth.

**Comment by dharma-guardian** [comment]
Ah, and there was the problem with the sort, which belongs to retrieval, which then lead to my misconception, ok.
